File: C Programs/Problems on String/Accept string & count number of capital characters.c
/* Write a program which accept string from user and count number of capital characters.

Input : “Marvellous Multi OS”
Output : 4

*/

#include<stdio.h>

int CountCapital(char *str)
{

    int iCnt=0;

    while(*str!='\0')
    {
            if((*str>='A') && (*str<='Z'))
            {
                iCnt++;
            }
            str++;
    }
     return iCnt;
}

int main()
{
    char arr[30];
    int iRet = 0;
    int iCnt=0;

    printf("Enter string: \n");
    scanf("%[^'\n']s",arr);

    iRet = CountCapital(arr);

    printf("Number of capital characters are: %d",iRet);

    return 0;
}

File: Java Programs/Problems on String/Accept string & count number of capital characters.java
/* Write a Java program which accept string from user and count number of capital characters.

Input : Marvellous Multi OS
Output : 4 

*/

import java.lang.*;
import java.util.*;

class StringDemo
{
    int CountCapital(String str)
    {
        char arr[]=str.toCharArray();
        int size=arr.length;
        int i=0;
        int iCnt=0;

        while(i<size)
        {
            if((arr[i]>='A')&&(arr[i]<='Z'))
            {
                iCnt++;
            }
            i++;
        }
        return iCnt;
    }
}

public class Demo
{
   public static void main(String arg[])
   {
       Scanner sobj=new Scanner(System.in);
       
       System.out.println("Enter String: ");
       String str=sobj.nextLine();
       
       StringDemo dobj=new StringDemo();
       
       int ret=dobj.CountCapital(str);
       System.out.println("No of capital characters are: "+ret);
   }
}
